src/displayapp/screens/Dice.cpp

```diff
@@ -0,0 +1,199 @@
+#include "displayapp/screens/Dice.h"
+#include "displayapp/screens/Screen.h"
+#include "displayapp/screens/Symbols.h"
+#include "components/settings/Settings.h"
+#include "components/motor/MotorController.h"
+#include "components/motion/MotionController.h"
+
+using namespace Pinetime::Applications::Screens;
+
+namespace {
+  static lv_obj_t* MakeLabel(lv_font_t* font,
```

Comment:
Functions in anonymous namespaces don't need to be declared as static, as they already have static linkage by default.

```suggestion
  lv_obj_t* MakeLabel(lv_font_t* font,
```